bin/cqlsh.py

```diff
@@ -600,6 +601,12 @@ def show_version(self):
     def show_session(self, sessionid, partial_session=False):
         print_trace_session(self, self.session, sessionid, partial_session)
 
+    def show_replicas(self, token, keyspace=None):
+        ks = self.current_keyspace if keyspace is None else keyspace
+        nodes = self.conn.metadata.token_map.get_replicas(self.current_keyspace, Murmur3Token(token))
+        addresses = [x.address for x in nodes]
+        print(f"Replicas for token {token} are {addresses}")
```

Comment:
I do not think we need `Replicas for token ... are ...`. That is just redundant information. It is also harder to parse in scripts if we use `cqlsh -e 'show replicas 95'`.